plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/eula/EULAUtils.java

```diff
@@ -51,4 +56,36 @@ public static String getPackageEula() {
         return eula;
     }
 
+    /*
+     *  License manager overrides all other workbench advisors if shown. We need to call EULA manually from it.
+     */
```

Comment:
Let's not include this comment here. Instead, you can move it to where this method is called.